### PR TITLE
test: add check status history tests

### PR DESCRIPTION
**Pull request by karel-rehor**
* Adds check status history tests
   * Slight refactoring of previous checks tests to reuse artifacts
   * Add some new data-testids to shared components

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 58 | :red_circle:
accessibility | 97 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=3196)

File: cypress/e2e/shared/checks.test.ts [hunk 1]
```diff
@@ -574,43 +575,133 @@ describe('Checks', () => {
...
+    } else {
+      throw `org ${org.name} has no id`
+    }
+    // get default org and bucket
+    // create check
+    return cy.createCheck(check).then((resp: any) => {
+      cy.wrap(resp.body).as(alias)
+    })
+    //        })
+    //      })
```
Comment (genehynson) [inline cypress/e2e/shared/checks.test.ts:700]: lets remove these

File: cypress/e2e/shared/checks.test.ts [hunk 2]
```diff
@@ -623,7 +714,8 @@ describe('Checks', () => {
           return cy.get<Organization>('@org').then((org: Organization) => {
             return cy.get<Bucket>('@bucket').then((bucket: Bucket) => {
               bucketName = bucket.name
-              if (
+              createCheck(check, org, bucket, 'check')
+              /* if (
```
Comment (genehynson) [inline cypress/e2e/shared/checks.test.ts:718]: some more commented code here to remove

File: cypress/e2e/shared/checks.test.ts [hunk 3]
```diff
@@ -737,4 +829,316 @@ describe('Checks', () => {
...
+                    writeCheckStatusRecord({
+                      check: thresult,
+                      sourceTimeStamp: status.sourceTimeStamp,
+                      level: status.level,
+                      message: status.message,
+                      pair: status.pair,
+                      offset: status.offset,
+                    })
+                  })
+                  cy.reload()
```
Comment (genehynson) [inline cypress/e2e/shared/checks.test.ts:983]: I'm a little nervous about this code block. `writeCheckStatusRecord` performs an async task but we're not waiting on the completion of that work here. I'm afraid that `cy.reload()` will get invoked before all of the `writeCheckStatusRecord` tasks are completed. And `cy.reload()` will terminate any pending API requests. Could we refactor this to add some `.then(() =>` chains?

**Review by genehynson — COMMENTED** [review]
overall looks good. Just a couple comments to remove and I left a concern about one of the code blocks

File: cypress/e2e/shared/checks.test.ts (hunk 1 above)
Comment (karel-rehor) [reply cypress/e2e/shared/checks.test.ts]: removed

File: cypress/e2e/shared/checks.test.ts (hunk 2 above)
Comment (karel-rehor) [reply cypress/e2e/shared/checks.test.ts]: removed

File: cypress/e2e/shared/checks.test.ts (hunk 3 above)
Comment (karel-rehor) [reply cypress/e2e/shared/checks.test.ts]: After playing around a bit I discovered that the command `writeLPData()` wraps a 'success' message on a successful return from `/api/v2/write`.  I set the call to `writeCheckStatusRecord()` to then assert that this message is returned.

**Review by genehynson — APPROVED** [review]
lgtm, thanks!

**Pull request merged**